spec/src/main/asciidoc/core/events.asciidoc

```diff
@@ -576,6 +576,7 @@ void afterLogin(@Observes @Priority(APPLICATION) LoggedInEvent event) { ... }
 
 The order of more than one observer with the same priority is undefined and the observer methods are notified therefore in a non predictable order.
 
+If a `@Priority` annotation is set on an asynchronous observer non-portable behaviour results.
```

Comment:
And what about:
If a `@Priority` annotation is declared on an event parameter of an asynchronous observer method, non-portable behavior results.

I would put this sentence next to `The priority of an observer method may be declared using the @Priority annotation.`, line 568. And maybe change the first sentence to `The priority of a synchronous observer method may be declared by annotating the event parameter with @Priority.`